test/cpp/qps/timer.h

```diff
@@ -42,6 +42,10 @@ class Timer {
     double wall;
     double user;
     double system;
+
+    double malloc_calls;
+    double mutex_locks;
+    double cv_waits;
   };
```

Comment:
Is double the right choice here? Since this is the C++ code, do we have a 64-bit int type available?